app/ide-desktop/lib/dashboard/src/authentication/src/index.tsx

```diff
@@ -0,0 +1,40 @@
+/** @file Authentication module used by Enso IDE & Cloud.
+ *
+ * This module declares the main DOM structure for the authentication/dashboard app. */
+/** This import is unused in this file, but React doesn't work without it, under Electron. This is
+ * probably because it gets tree-shaken out of the bundle, so we need to explicitly import it. */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import * as React from "react";
+import * as reactDOM from "react-dom/client";
```

Comment:
all the react imports are `as React` - so should this be `as ReactDOM`?

Reply:
I prefer `pascalCase` for the imports where possible, but in the `index.tsx` `import * as React from "react";` has a special purpose. Without that import, `React` doesn't get included in the Electron bundle for some reason. Or if the import is changed to `import * as react` rather than `import * as React` we get the same thing. So this is why this one import alone is unstandardized.

I'll document this quirk.

Reply:
does that mean `React` can be `react` in all files other than `index.tsx` though?

Reply:
Yes, it can. I thought that was already the case but I'll double check.

Reply:
hmm, maybe i was looking at a different PR